34.md

```diff
@@ -96,6 +94,9 @@ The first patch revision in a patch revision SHOULD include a NIP-10 `e` `reply`
     ["parent-commit", "<parent-commit-id>"],
     ["commit-pgp-sig", "-----BEGIN PGP SIGNATURE-----..."], // empty string for unsigned commit
     ["committer", "<name>", "<email>", "<timestamp>", "<timezone offset in minutes>"],
+
+    ["branch-name", "<custom-branch-name>"], // optional short name for use by git-remote-nostr
```

Comment:
Is this just an implementation detail of git-remote-nostr? If yes then maybe we shouldn't include it here.